cmd/boulder-sa/main.go

```diff
@@ -25,7 +25,7 @@ func main() {
 		dbMap, err := sa.NewDbMap(dbURL)
 		cmd.FailOnError(err, "Couldn't connect to SA database")
 
-		sai, err := sa.NewSQLStorageAuthority(dbMap, clock.Default())
```

Comment:
Nice config gating! But I think we actually don't need it: migrations are run before deploys, so the table should be ready by the time the new code runs.